Src/Fido2/IFido2.cs

```diff
@@ -20,12 +20,14 @@ Task<VerifyAssertionResult> MakeAssertionAsync(
         IReadOnlyList<byte[]> storedDevicePublicKeys,
         uint storedSignatureCounter,
         IsUserHandleOwnerOfCredentialIdAsync isUserHandleOwnerOfCredentialIdCallback,
+        byte[]? requestTokenBindingId = null,
         CancellationToken cancellationToken = default);
 
     Task<RegisteredPublicKeyCredential> MakeNewCredentialAsync(
         AuthenticatorAttestationRawResponse attestationResponse,
         CredentialCreateOptions originalOptions,
         IsCredentialIdUniqueToUserAsyncDelegate isCredentialIdUniqueToUser,
+        byte[]? requestTokenBindingId = null,
         CancellationToken cancellationToken = default);

```

Comment:
Consider making a new object for the parameters so we can continue to add or remove parameters in the future without a binary breaking change.

```csharp
public class MakeNewCredentialRequest
{    
    public required AuthenticatorAttestationRawResponse AttestationResponse { get; init; }
    
    public required CredentialCreateOptions OriginalOptions { get; init; }
    
    // for Fido2 2024 conformance testing only
    [EditorBrowsable(EditorBrowsableState.Never)] 
    public byte[]? RequestTokenBindingId { get; init; }
}
```